CODE_OF_CONDUCT.md

```diff
@@ -1,9 +1,9 @@
 ## Code of Conduct
 
-BeeWare operates under a [Code of Conduct](https://pybee.org/community/behavior/code-of-conduct/)
+BeeWare operates under a [Code of Conduct](https://beewareorg/community/behavior/code-of-conduct/)
```

Comment:
Typo on beeware.org here

Reply:
Thanks for this feedback, have corrected the error in next commit.